app/models/user.rb

```diff
@@ -62,11 +62,11 @@ def all_listings
   end
 
   def lists_except_movie(movie = nil)
-    if movie.present? && Movie.exists?(tmdb_id: movie.tmdb_id)
-      @movie_lists = Movie.find_by(tmdb_id: movie.tmdb_id).lists.by_user(self)
-      @lists_except_movie = (all_lists - @movie_lists)
+    if movie.present? && existing_movie = Movie.find_by(tmdb_id: movie.tmdb_id)
+      (all_lists - existing_movie.lists.by_user(self))
+        .sort_by { |list| list.name.downcase }
```

Comment:
The `sort_by` bit is the only actual change. the other stuff was just refactoring.

The `sort_by` is needed because of the existing jank setup that is doing two different queries (checking a lists's owner, as well as any lists the user is a _member_ of). It returns an array, not an ActiveRecord relationship, hence the need to sort it this way.

This can be improved by upgrading to Rails 5 and using the ActiveRecord `or`. But probably a better option is to _always_ use the `memberships` join table (links users to lists), which would simplify a lot. I think. I'd have to look more into it.

Reply:
Is `existing_movie = Movie.find_by` supposed to be `=` or `==`? If `=`, is there any chance we can move the assignment of `existing_movie` into another method? Something like 
```
def existing_movie(movie)
  existing_movie = Movie.find_by(tmdb_id: movie.tmdb_id)
end
```